composer.json

```diff
@@ -36,7 +36,7 @@
   "require": {
     "php": ">=5.4.0",
     "phpseclib/phpseclib": "2.0.*",
-    "vlucas/phpdotenv": ">=2.4.0 <=2.6"
+    "vlucas/phpdotenv": "<3.0"
```

Comment:
Let's keep the min version as well. With some projects we might end up getting really old versions of dotenv.

Reply:
Good point, updated.